src/client/Microsoft.Identity.Client/LogLevel.cs

```diff
@@ -9,6 +9,11 @@ namespace Microsoft.Identity.Client
     /// </summary>
     public enum LogLevel
     {
+        /// <summary>
+        ///     LogAlways Log Level
+        /// </summary>
+        LogAlways = -1,
```

Comment:
Nit. Wondering if we could make the wording more consistent here and throughout? (I understand we're matching AzureSDK here.) Ex.: Essential, vital, required, fundamental? Or maybe just remove `Log` prefix so it's `LogLevel.Always`.

I'm ok either way though because Essential describes the kind of data is logged, Always describes when it is logged.

Reply:
I think `LogLevel.Always` makes sense